client/src/RoutingContainer.tsx

```diff
@@ -1,21 +1,22 @@
-import React, { FC } from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
-import ScrollToTop from './components/ScrollToTop';
-import CanvasPageContainer from './canvas/containers/CanvasPageContainer';
-import AboutPage from './about/components/About';
-import Header from './header/containers/HeaderContainer';
-import Footer from './footer/components/Footer';
+import React, { FC } from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import ScrollToTop from "./components/ScrollToTop";
+import CanvasPageContainer from "./canvas/containers/CanvasPageContainer";
+import AboutPage from "./about/components/About";
+import Header from "./header/containers/HeaderContainer";
+import Footer from "./footer/components/Footer";
+import ColorPicker from "./colorPicker/colorPicker";
 
 const RoutingContainer: FC = () => (
   <Router>
     <ScrollToTop>
       <Header />
+      <ColorPicker />
```

Comment:
Could we put this in `CanvasPageContainer`? Typically we only want to put pages in the routing container